src/metrics.ts

```diff
@@ -177,13 +183,18 @@ export const MetricsProcessor = (
   };
 
   const _send = (): void => {
+    if (closed) {
```

Comment:
I noticed that if any evaluations were made after the SDK had been closed, we'd still process and send them. This stops that from happening.

Reply:
It won't stop them being queued - but this is the easier win.